src/main/java/net/sf/mzmine/modules/tools/msmsspectramerge/MzMergeMode.java

```diff
@@ -0,0 +1,73 @@
+package net.sf.mzmine.modules.tools.msmsspectramerge;
+
+import net.sf.mzmine.datamodel.DataPoint;
+import net.sf.mzmine.util.scans.ScanUtils;
+
+import java.util.Arrays;
+
+public interface MzMergeMode {
```

Comment:
Does this make sense as an interface? Shouldn't it be an enum with some accompanying class to provide the functionality?

Reply:
I don't see a difference between enums and singletons, nor do I see any advantage of enums

Reply:
Readability. _interface_ is for defining interfaces, _enum_ is for enumerating options. I'm not arguing about technical differences, but readability is a very important aspect in long-term, particularly for a collaborative project like MZmine.

Reply:
I still don't see why a switch statement is in any kind more readable than an interface. These are operations, functions, not options. If you don't like to have a function is used as parameter in the dialogue we can add another enum class which points to the corresponding functions. Makes things more complicated though.

Reply:
Maybe I misunderstood your intent here. If this interface defines different merging functions, then it's fine.. perhaps my choice of name would be MzMergeFunction rather than MzMergeMode, but that's a detail.